kong/include/kong/model/config.proto

```diff
@@ -32,4 +32,6 @@ message Config {
 
   repeated Workspace workspaces = 12;
   repeated Parameter parameters = 13;
+
+  repeated KeyAuth keyauth_credentials = 14;
```

Comment:
let's give it some room for other potential core entities maybe let keyauth_credentials start from 100 as well; on the other side, let's add other known custom DAO entities here as well.